GameOfLife/View/GameView.cs

```diff
@@ -1,31 +1,23 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace GameOfLife
+﻿namespace GameOfLife
 {
     /// <summary>
-    /// 
+    /// View of the game field.
     /// </summary>
     public class GameView
     {
         /// <summary>
-        /// Makes view of the game field
+        /// Makes view of the game field.
         /// </summary>
-        /// <param name="args">Array containing the game field</param>
+        /// <param name="args">Array containing the game field.</param>
         public void View(int[,] args)
         {
             Console.Clear();
-
-            for (int x = 0; x < args.GetLength(0); x++)
+            for (int row = 0; row < args.GetLength(0); row++)
```

Comment:
You changed variable name, but you should change retrieval of max range value:
for (int row = 0; row < args.GetLength(**1**); row++)
Make changes for column accordingly.

Reply:
My careless mistake. Fixed.